src/api/lib/tasks/polymorphic_comment.rake

```diff
@@ -0,0 +1,19 @@
+namespace :data do
+  desc "Moves the data related to the belongs_to relation in Comment to allow the old comments to work with the polymorphic relation."
+  task polymorphic_comment: :environment do
+    puts "Moving the data of Comments to allow the old comments to work with the polymorphic relation"
+    Comment.inheritance_column = nil
+    moved_comments = 0
+    Comment.where("type IS NOT NULL").find_each(batch_size: 5000) do |c|
+      /\AComment(?<type>.+)/ =~ c.type
+      if type
+        c.object_type = type
+        c.object_id = c.project_id || c.package_id || c.bs_request_id
+        c.type = c.project_id = c.package_id = c.bs_request_id = nil
+        moved_comments += 1 if c.save
+      end
+    end
+    Comment.inheritance_column = 'type'
```

Comment:
Before we set this we should check
```
Comment.where("type IS NOT NULL").count == moved_comments
```
And if not show a proper error message!
What do you think?